spec/spec_helper.rb

```diff
@@ -33,7 +33,7 @@
 require 'test_app'
 
 module Fixtures
-  TEST_CONFIG = Mutant::Config::DEFAULT.update(reporter: Mutant::Reporter::Trace.new)
+  TEST_CONFIG = Mutant::Config::DEFAULT.with(reporter: Mutant::Reporter::Trace.new)
```

Comment:
I know `#with` was added in Anima, but I wonder if maybe `#merge` would be a better alternative. It implies the object state is being merged with new state and that a new object is being returned. The `#with` method isn't commonly used for this purpose.